packages/content/test/attributes/HouseNumber.test.ts

```diff
@@ -0,0 +1,63 @@
+import { ParsingError } from "@js-soft/ts-serval";
+import { CountryAlpha2, StreetAddress } from "../../src";
+
+describe("creation of RepositoryAttributes of Attribute Value Type Streetadress", function () {
+    test("can create a RepositoryAttribute of Attribute Value Type Streetadress", function () {
+        const validStreetAdress = StreetAddress.from({
+            recipient: "Hugo Becker",
+            street: "Luisenstr.",
+            houseNo: "38-40",
+            zipCode: "76646",
+            city: "Bruchsal",
+            country: CountryAlpha2.DE,
+            state: "Baden-Württemberg"
+        });
+        expect(validStreetAdress.recipient.toString()).toBe("Hugo Becker");
+        expect(validStreetAdress.street.toString()).toBe("Luisenstr.");
+        expect(validStreetAdress.houseNo.toString()).toBe("38-40");
+        expect(validStreetAdress.zipCode.toString()).toBe("76646");
+        expect(validStreetAdress.city.toString()).toBe("Bruchsal");
+        expect(validStreetAdress.country.toString()).toBe(CountryAlpha2.DE);
+        expect(validStreetAdress.state?.toString()).toBe("Baden-Württemberg");
+    });
+    test("returns an error when trying to create an Attribute Value Type Streetadress with an invalid value for houseNo.", function () {
```

Comment:
I'd write in the test's name briefly more precisely why the value is invalid (houseNo consists of letters only).

Same for the other test below.